rotkehlchen/premium/sync.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, migration_manager: DataMigrationManager, data: DataHandler) -
             self.last_data_upload_ts = data.db.get_cache(
                 cursor=cursor, name=DBCache.LAST_DATA_UPLOAD_TS,
             )
-            self.last_upload_attempt_ts = self.last_data_upload_ts
+            self.last_upload_attempt_ts = 0 if self.last_data_upload_ts is None else self.last_data_upload_ts  # noqa: E501
```

Comment:
Shouldn't this be a `timestamp(0)`? Does it make sense to add a default to the get_cache method so you can provide the default value to return if none is found?

Reply:
This was the case before. But now we are also adding `used_query_ranges` values, which are checked to be `None` in some places. That's why I'm changing the default of `get_cache()` from `0` to `None`. Also, in the future, we may store strings in the table.

Reply:
But yeah, updating this one to `Timestamp(0)`.